mangaki/mangaki/wrappers/anilist.py

```diff
@@ -25,14 +25,163 @@ def to_python_datetime(date):
      ...
     ValueError: no valid date format found for 2015
     """
-    date = date.strip()
+    date = str(date).strip()
     for fmt in ('%Y%m%d', '%Y%m00', '%Y0000'):
         try:
             return datetime.strptime(date, fmt)
         except ValueError:
             pass
     raise ValueError('no valid date format found for {}'.format(date))
 
+def to_anime_season(date):
+    """
+    Return the season corresponding to a date
+    >>> to_anime_season(datetime.datetime(2017, 3, 3, 0, 0))
+    'winter'
+    """
+    if 1 <= date.month <= 3:
+        return 'winter'
+    elif 4 <= date.month <= 6:
+        return 'spring'
+    elif 7 <= date.month <= 9:
+        return 'summer'
+    else:
+        return 'fall'
+
+
+class AniListWorks(Enum):
+    animes = 'anime'
+    mangas = 'manga'
+
+
+class AniListStatus(Enum):
+    aired = 'finished airing'
+    airing = 'currently airing'
+    published = 'finished publishing'
+    publishing = 'publishing'
+    anime_coming = 'not yet aired'
+    manga_coming = 'not yet published'
+    cancelled = 'cancelled'
+
+
+ANILIST_MEDIA_TYPES = ['TV',  'TV Short', 'Movie', 'Special', 'OVA', 'ONA', 'Music',
+                       'Manga', 'Novel', 'One Shot', 'Doujin', 'Manhua', 'Manhwa']
+
+
+class AniListMediaType(Enum):
+    tv = 0
```

Comment:
Is there a particular reason to use a list `ANILIST_MEDIA_TYPES` and an Enum when you can just embed the string into the Enum rather than the indexes ?